configure.ac

```diff
@@ -545,6 +545,22 @@ typedef unsigned __int32 uint32_t;
     fi
     ;;
 esac
+AC_CACHE_CHECK([volatile asm support],
+               [je_cv_volatile_asm],
+               AC_RUN_IFELSE([AC_LANG_PROGRAM(
+[[
+]],
+[[
+      void* ptr;
+      asm volatile("" : "+r"(ptr));
+      return 0;
+]])],
+[je_cv_volatile_asm=yes],
+[je_cv_volatile_asm=no],
+[je_cv_volatile_asm=no]))
+if test "x${je_cv_volatile_asm}" = "xyes"; then
+  AC_DEFINE([JEMALLOC_HAVE_VOLATILE_ASM], 1, [])
```

Comment:
nit: `AC_DEFINE([JEMALLOC_HAVE_VOLATILE_ASM], [ ], [ ])`. The value isn't necessary; also note that there should be a space inside the brackets.

Reply:
Modified, also update `rdtscp` in teh same pattern: `AC_DEFINE([JEMALLOC_HAVE_RDTSCP], [ ], [ ])`